src/node.cc

```diff
@@ -138,4 +207,31 @@ std::vector<std::tuple<std::string, Port>> Node::oports() const {
     return oports;
 }
 
+void Node::set_dynamic_port(Port port) {
```

Comment:
Can we introduce a generic method to be able to connect ports after constructing the graph?

Reply:
```
void Node::set_iport(Port port)
void Node::set_oport(Port port)
```